cmake/generate_sbom.cmake

```diff
@@ -277,6 +355,15 @@ FUNCTION(GENERATE_SBOM)
     IF(cpe)
       SET(cpe "\n      \"cpe\": \"${cpe}\",")
     ENDIF()
+
+    GET_YAML_VALUE(THIRD_PARTY "${repo_name_lower}.license" license)
+    IF(NOT license)
+      MESSAGE(FATAL_ERROR "no license for 3rd party dependency ${repo_name_lower}.")
+    ENDIF()
+    GET_YAML_VALUE(THIRD_PARTY "${repo_name_lower}.copyright" copyright)
+    IF(NOT copyright)
+      SET(copyright NOASSERTION)
+    ENDIF()
```

Comment:
did you double-check that it's the correct behavior? a license must be present, a copyright is optional?

Reply:
No, just experience. Every project we depend on has a license, while most projects do not have a copyright statement. We did not have, until 2 weeks ago.